utils/composer-packages/src/Command/AddComposerTypo3ExtensionsToConfigCommand.php

```diff
@@ -44,8 +43,7 @@ public function __construct(
 
     protected function configure(): void
     {
-        $this->setName(CommandNaming::classToName(self::class));
-        $this->setAliases(['typo3-extensions']);
+        $this->setName('typo3-extensions');
         $this->setDescription('[DEV] Add TYPO3 extensions from packagist.org to composer configuration');
```

Comment:
At first it seemed like a great idea, but I learned the command naming is now too magic and hard to read in the code.
The `typo3-extensions` is much more clear :)